gsm.yaml

```diff
@@ -0,0 +1,74 @@
+package:
+  name: gsm
+  version: 1.0.22
+  epoch: 0
+  description: Shared libraries for GSM speech compressor
+  copyright:
+    - license: TU-Berlin-2.0
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - automake
+      - autoconf
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha512: f4284f096ed4d606ddece5c1b28edb63feacc26a0b664f1350c040dbf50113d27bedfd9896bf1620cb4fc3a6c8f6a5244955a6c625ded833dcee607c63c97dd2
+      uri: https://www.quut.com/gsm/gsm-${{package.version}}.tar.gz
+      strip-components: 1
+
+  - uses: patch
+    with:
+      patches: gsm-64bit.patch
+
+  - uses: patch
+    with:
+      patches: gsm-makefile.patch
```

Comment:
Where do these patches come from? I'd love to understand what they're doing a bit more, or at least have a pointer if things go wrong.

Reply:
https://git.alpinelinux.org/aports/tree/main/gsm